src/StructuredLogViewer.Avalonia/FileTypes.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Avalonia.Platform.Storage;
+
+namespace StructuredLogViewer.Avalonia;
+
+public class FileTypes
+{
+    public static string BinlogDefaultExtension = ".binlog";
+
+    public static FilePickerFileType Binlog { get; } =
+        new("Binary Log")
+        {
+            Patterns = new[] { "*.binlog", "*.buildlog" },
+            MimeTypes = new[] { "application/binlog", "application/buildlog" },

```

Comment:
Currently Browser backend won't understand these custom mime types. That's something to be fixed on the avalonia side.